posts/epi-community-contrib/index.qmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "Epiverse community engagement and software sustainability for research software"
+author:
+  - name: "Joshua W. Lambert"
+    orcid: "0000-0001-5218-3046"
+date: "2025-07-22"
+categories: [open-source, R, R package, epidemiology, community, Epiverse, DOI]
+bibliography: index.bib
+format:
+  html:
+    toc: true
+---
+
+Software that is developed for research or by researchers can be difficult to maintain given the incentive and funding structures in academia. This remains true for epidemiology, with a large volume of software written during the COVID-19 pandemic, much of which is now abandonware[^1]. This does not mean that the software developed to understand the COVID-19 pandemic was bad or does not have utility in understanding future epidemics and pandemics, but just that the capacity to maintain and further develop these tools is not available now the pandemic is no [longer considered an acute public health emergency](https://www.who.int/news/item/05-05-2023-statement-on-the-fifteenth-meeting-of-the-international-health-regulations-(2005)-emergency-committee-regarding-the-coronavirus-disease-(covid-19)-pandemic).
+
+These issues around software sustainability and the academic structures that hinder software longevity were raised by @kucharskiCOVID19ResponseIllustrates2020 and were one of the leading reasons for the [Epiverse-TRACE initiative](https://epiverse-trace.github.io/). Alongside the developing novel software (R packages), Epiverse also has a commitment to support the community of package developers in epidemiology and outbreak analytics.
```

Comment:
Last sentence bothers me mildly; maybe more like `One of the key Epiverse objectives is "[pull quote from proposal]".`? Not sure what that pull quote looks like, but the current wording here is a bit awkward. I also don't think its necessary to invoke that y'all also mint new software.

Reply:
I've gone through some of the early documentation for the project proposal and couldn't find any information that would fit into this context. Related is the mention of the [CSCCE Community Participation Model](https://www.cscce.org/resources/cpm/). 

> I also don't think its necessary to invoke that y'all also mint new software.

I wanted to phrase it this way as I didn't want it come across as contributing to other software is our primary activity. (Added bonus for advertising our tools).